.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue is stale because it has been open 365 days with no activity.'
+        stale-pr-message: 'This PR is stale because it has been open 365 days with no activity.'
+        stale-issue-label: 'Stale-Issue'
+        stale-pr-label: 'Stale-PR'
+        exempt-pr-label: 'Not-Stale'
```

Comment:
any suggestions for better label name? :smiley:

Reply:
* For a stale label, I'd suggest "Status: Stale" to match our existing categories (we haven't been using these categories very systematically, but that's a separate discussion)
* Probably OK to use the same label for both issues and PRs
* Rather than a specific non-stale label, I'd exempt the categories whose issues are likely to have extra-long lifecycles, e.g. if we already had "wishlist" or "backlog" labels I'd nominate them to be exempt from stale markers. But on reviewing the labels we *do* have, I'm not sure which ones need exemption. Anyone else want to weigh in here?

Reply:
@infotroph Right now, we can specify a [single exempt label](https://github.com/actions/stale/issues/17) :disappointed: only.